js/articleView.js

```diff
@@ -1,75 +1,84 @@
-//  Configure a view object, to hold all our functions for dynamic updates and article-related event handlers.
-var articleView = {};
+(function(module) {
 
-articleView.handleAuthorFilter = function() {
-  $('#author-filter').on('change', function() {
-    if ($(this).val()) {
-      console.log($(this).val());
-      $('article').hide();
-      $('article[data-author="' + $(this).val() + '"]').fadeIn();
-    } else {
-      $('articles').fadeIn();
-      $('article.template').hide();
-    }
-    $('#category-filter').val('');
-  });
-};
+  //  CONFIGURE A VIEW OBJECT TO HOLD ALL FUNCTIONS FOR DYNAMIC UPDATES AND ARTICLE-RELATED EVENT HANDLERS.
+  var articleView = {};
 
-articleView.handleCategoryFilter = function() {
-  $('#category-filter').on('change', function() {
-    if ($(this).val()) {
-      console.log($(this).val());
-      $('article').hide();
-      $('article[data-category="' + $(this).val() + '"]').fadeIn();
-    } else {
-      $('articles').fadeIn();
-      $('article.template').hide();
-    }
-    $('#author-filter').val('');
-  });
-};
+  articleView.handleAuthorFilter = function() {
+    $('#author-filter').on('change', function() {
+      if ($(this).val()) {
+        console.log($(this).val());
+        $('article').hide();
+        $('article[data-author="' + $(this).val() + '"]').fadeIn();
+      } else {
+        $('articles').fadeIn();
+        $('article.template').hide();
+      }
+      $('#category-filter').val('');
+    });
+  };
 
-articleView.handleMainNav = function () {
-  console.log('into handleMainNav');
-  $('.main-nav').on('click', '.tab', function(e) {
-    $('.tab-content').hide();
-    $('#' + $(this).data('content')).fadeIn();
-  });
-  $('.main-nav .tab:first').click();
-};
+  articleView.handleCategoryFilter = function() {
+    $('#category-filter').on('change', function() {
+      if ($(this).val()) {
+        console.log($(this).val());
+        $('article').hide();
+        $('article[data-category="' + $(this).val() + '"]').fadeIn();
+      } else {
+        $('articles').fadeIn();
+        $('article.template').hide();
+      }
+      $('#author-filter').val('');
+    });
+  };
 
```

Comment:
I am having a similar problem on my own portfolio page with the category filters. Since implementing handlebars they are not working properly. At the moment they are not really necessary because of the amount of articles but as you continue to add features they could become very helpful. If you do decide to keep them I would consider adjusting the author filter to something else since you will most likely be the only author. I filtered mine (before it broke) by title and category but again was thinking that filtering by certain skills would be a good idea i.e. css, html, javascript, jquery etc.